community/lexicon/locations/address.json

```diff
@@ -0,0 +1,46 @@
+{
+    "lexicon": 1,
+    "id": "community.lexicon.locations.address",
+    "defs": {
+        "main": {
+            "type": "object",
+            "description": "A physical location in the form of an address.",
+            "required": [
+                "country"
+            ],
+            "properties": {
+                "country": {
+                    "type": "string",
+                    "description": "The ISO 3166 country code. Preferably the 2-letter code.",
+                    "minLength": 2,
+                    "maxLength": 10
+                },
+                "postalCode": {
+                    "type": "string",
+                    "description": "The postal code of the location.",
+                    "maxLength": 20
+                },
+                "region": {
+                    "type": "string",
+                    "description": "The administrative region of the country. For example, a state in the USA.",
+                    "maxLength": 100
+                },
+                "locality": {
+                    "type": "string",
+                    "description": "The locality of the region. For example, a city in the USA.",
+                    "maxLength": 100
+                },
+                "street": {
```

Comment:
Most address forms I encounter have 2 street fields. It must be common enough if they are all doing it. We should probably do similar

Reply:
Alternatively, people can just use newlines inside this field.

Reply:
100 chars is likely insufficient for doing that

Reply:
there ought to be ample test data out there we can use to inform our decsisions